example/k3d/scripts/smoke-test.bash

```diff
@@ -37,6 +37,7 @@ ROOT=$(git rev-parse --show-toplevel)
 K3D_CLUSTER_NAME="agent-smoke-test"
 MUTATION_FREQUENCY="5m"
 CHAOS_FREQUENCY="30m"
```

Comment:
these end up being unused at this point, need to figure out how to pass these values during tk apply if that's possible

Reply:
Yep: `tk apply <path> -V key1=value1 -V key2=value2`

Then in jsonnet: `std.extVar("key1")`